app/src/main/java/org/apache/fineract/ui/online/loanaccounts/loanaccountlist/LoanAccountsFragment.java

```diff
@@ -53,6 +57,9 @@
     @BindView(R.id.layout_error)
     View layoutError;
 
+    @BindView(R.id.fab_add_customer_loan)
+    FloatingActionButton fabLoan;
```

Comment:
Please rename the view from `fabLoan` to `fabAddCustomerLoan`

Reply:
I will change it ASAP